src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md

```diff
@@ -2,7 +2,16 @@
 
 ## Unreleased
 
-## 1.2.5 [2022-Apr-20]
+## 1.2.6 [2022-Apr-21]
+
+* Set GenevaMetricExporter temporality preference back to Delta.
+[xxx](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/xxx)
```

Comment:
Update the PR#

Reply:
haha, yes! this two step process is easy to forget :D